pontoon/base/templates/google_analytics.html

```diff
@@ -0,0 +1,14 @@
+{% if settings.GOOGLE_ANALYTICS_KEY %}
```

Comment:
I'd rename this file to `analytics`, or even better, `tracker.html` so that in the future, if we're to change tracker, we don't have to rename.

Reply:
Done :) I decided to rebase the pull request because it's a small change and it's still merge-able.